engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/DateTest.scala

```diff
@@ -0,0 +1,57 @@
+package org.enso.interpreter.test.semantic
+
+import org.enso.interpreter.test.{InterpreterContext, InterpreterTest}
+
+class DateTest extends InterpreterTest {
+  override def subject: String = "Debug.eval"
+
+  override def specify(implicit
+    interpreterContext: InterpreterContext
+  ): Unit = {
+    "evaluate a date expression" in {
+      val code =
+        s"""from Standard.Builtins import all
+           |
+           |import Standard.Base.Data.Time.Date
+           |
+           |main =
+           |    Debug.eval $rawTQ
```

Comment:
Why the `Debug.eval` here?
Also, I'd rather we place these in `tests/Test` – this here is a test suite for very low-level stuff, mostly used to bootstrap the language to the point of it being able to test itself. I'd even hesitantly call it "deprecated" now.

Reply:
> I admit, I just copied EvalTest.scala and it uses Debug.eval for some reason.

The reason is `EvalTest` is testing the `Debug.eval` method.

> Tests for org.graalvm.polyglot usage deserve to be written in Java, I assume. Do we have such tests somewhere?

To me, they test the functionality of Enso (i.e. "Enso is a language that can share values with other languages") – as evidenced by the fact that you haven't written a single line of non-trivial java in this test – just using it to eval Enso code and compare strings.

The best place to drop these would be [`Java_Interop_Spec`](https://github.com/enso-org/enso/blob/develop/test/Tests/src/Semantic/Java_Interop_Spec.enso).

Reply:
Done in [6267ba2](https://github.com/enso-org/enso/pull/3374/commits/6267ba20f4a34ca84ca97c0d8f2730ec5b376c28). Running:
```bash
$ sbt --java-home /graalvm-ce-java11-21.3.0/ buildEngineDistribution
$ ./distribution/bin/enso --run ./test/Tests/src/Semantic/Java_Interop_Spec.enso
```
should fail the test. After doing the same on the merge commit [dd15acf](https://github.com/enso-org/enso/pull/3374/commits/dd15acfadcf0d4cc95583f8b216a68679e8bb4f1) the test shall succeed.